release/_releases_json.sh

```diff
@@ -1,6 +1,21 @@
 #!/bin/bash
 
+function _add_new_version {
+	echo "Add ${LIFERAY_RELEASE_VERSION} version to supported-${LIFERAY_RELEASE_PRODUCT_NAME}-versions.txt"
```

Comment:
@Wykenz, please use `lc_log INFO` instead of `echo`. See an example on line 33. :slightly_smiling_face: